Vault-Deployment/add-to-hosts.sh

```diff
@@ -22,10 +22,9 @@ echo -e "### vault playground start ###" | sudo tee -a /etc/hosts
 echo -e "#the cleanup script will delete everything between this tags" | sudo tee -a /etc/hosts
 
 
-add_ingress_to_hosts "vault-ui-ingress" "vault" "vault-ui.playground.lab"
-add_ingress_to_hosts "vault-cli-ingress" "vault" "vault-cli.playground.lab"
-add_ingress_to_hosts "prometheus-ingress" "$VAULT_K8S_NAMESPACE" "prometheus.playground.lab"
-add_ingress_to_hosts "grafana-ingress" "$VAULT_K8S_NAMESPACE" "grafana.playground.lab"
+add_ingress_to_hosts "vault" "vault" "vault.playground.lab"
```

Comment:
The changes here a part of the change that removed the Terraform code for the Ingress (and only maintain one name for Vault API). The Helm/yaml deployment approach has slightly different Ingress names, so that's what changed here.